controller.go

```diff
@@ -547,6 +563,11 @@ func (c *controller) Sandboxes() []Sandbox {
 
 	list := make([]Sandbox, 0, len(c.sandboxes))
 	for _, s := range c.sandboxes {
+		// Hide stub sandboxes from libnetwork users
+		if s.isStub {
+			continue
+		}
```

Comment:
thanks. I thought the existing sandboxCleanup during the daemon bootup addressed all the cases for locally scoped networks. I guess with this PR, it seems there are some left over sandboxes (for local scoped networks) even after the sandboxCleanup function. Is there a particular case that triggers this scenario ?